test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -170,7 +170,7 @@ def test_ignored_modules_invalid(self):
             expected_warning=UserWarning,
             expected_regex="Trying to ignore the top-level module passed into "
             "the FSDP constructor itself will result in all parameters being "
-            "ignored and is not supported",
+            "ignored",
```

Comment:
curious why are we removing not supported language?

Reply:
I changed the message in this PR to "and is not well-supported" instead of "and is not supported" because execution can still work. It is just not meaningful: There will be a top-level-only FSDP with no `FlatParameter`s. I am not sure what the point of that is. I relaxed the regex in case we decide to change this phrasing in the future. The regex captures the important part of the message.